src/modules/vendor/pages/ni/management/BlendedCoursesDirectory.vue

```diff
@@ -92,9 +93,10 @@ export default {
       newCourse: {
         program: { required },
         subProgram: { required },
-        company: { required: requiredIf(this.newCourse.type === INTRA) },
+        company: { required: requiredIf(this.isIntraCourse) },
```

Comment:
est-ce-que de la même manière que pour maxTrainees il ne faudrait pas mettre ...(this.isIntraCourse && { company: { required } }), ? on veut avoir des formation avec company qui ne sont pas intra ?

Reply:
Non je ne pense pas que l'on veuille avoir des formations avec company, je fais la modif